src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="/" 
+      method="get"
+      data-qa="big"
+    >
+      <label class="label" for="search-bar-b">
+        <img 
+          class="search-icon icon-b" 
+          src="./images/Search.svg" 
+          alt="search-icon">
+        <input 
+          class="search-bar big" 
+          type="text" 
+          name="search-bar-b" 
+          id="search-bar-b"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </label>
+    </form>
+    <form 
+      action="/" 
+      method="get"
+      data-qa="small"
+    >
+      <label class="label" for="search-bar-s">
+        <img 
+          class="search-icon icon-s" 
+          src="./images/Search.svg" 
+          alt="search-icon">
+        <input 
+          class="search-bar small" 
+          type="text" 
+          name="search-bar-s" 
+          id="search-bar-s"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="small"
+        >
+      </label>
+    </form>
```

Comment:
```suggestion
    <form 
      action="/" 
      method="get"
      data-qa="big"
    >
      <label class="label" for="search-bar-b">
        <img 
          class="search-icon" 
          src="./images/Search.svg" 
          alt="search-icon">
        <input 
          class="search-bar" 
          type="text" 
          name="search-bar-b" 
          id="search-bar-b"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </label>
    </form>
    <form 
      action="/" 
      method="get"
      data-qa="small"
    >
      <label class="label" for="search-bar-s">
        <img 
          class="search-icon search-icon-small" 
          src="./images/Search.svg" 
          alt="search-icon">
        <input 
          class="search-bar search-bar-small" 
          type="text" 
          name="search-bar-s" 
          id="search-bar-s"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="small"
        >
      </label>
    </form>
```

lets make it work like this